lib/components/admin/call-history-window.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import * as callTakerActions from '../../actions/call-taker'
+import CallRecord from './call-record'
+import DraggableWindow from './draggable-window'
+import Icon from '../narrative/icon'
+import {WindowHeader} from './styled'
+
+class CallHistoryWindow extends Component {
+  render () {
```

Comment:
Since there is just this render method, this could be converted into a functional compnent.